benchmarking/runs/real_data/cosc_341/scenario1.py

```diff
@@ -0,0 +1,231 @@
+    """
+
+    def start(self, num_trials: int = 1, compute_metrics: bool = False):
+        scenario_1 = ConcentrateTimeSlotDiversifyGenderMin2AndDiversifyYearLevel()
+        scenario_2 = DiversifyGenderMin2ConcentrateTimeSlotAndDiversifyYearLevel()
+
+        metrics = {
+            "PrioritySatisfaction": PrioritySatisfaction(
+                goals_to_priorities(scenario_1.goals),
+                False,
+            ),
+            "AverageTimeslotCoverage": AverageTimeslotCoverage(
+                available_timeslots=[1, 2, 3, 4, 5, 6],
+            ),
+            "GenderCosineDifference": AverageCosineDifference(
+                name="GenderCosineDifference",
+                attribute_filter=[ScenarioAttribute.GENDER.value],
+            ),
+            "YearLevelCosineDifference": AverageCosineDifference(
+                name="YearLevelCosineDifference",
+                attribute_filter=[ScenarioAttribute.YEAR_LEVEL.value],
+            ),
+            "AverageSoloStatus": AverageSoloStatus(
+                minority_groups_map={
+                    ScenarioAttribute.GENDER.value: [Gender.FEMALE.value],
+                }
+            ),
+        }
+
+        student_provider = COSC341W2021T2AnsweredSurveysStudentProvider()
+        cache_key = "real_data/cosc_341/scenario1"
+        simulation_settings_1 = SimulationSettings(
+            num_teams=math.ceil(175 / self.TEAM_SIZE),
+            student_provider=student_provider,
+            scenario=scenario_1,
+            cache_key=cache_key,
+        )
+
+        simulation_settings_2 = SimulationSettings(
+            num_teams=math.ceil(175 / self.TEAM_SIZE),
+            student_provider=student_provider,
+            scenario=scenario_2,
+            cache_key=cache_key,
+        )
+
+        artifacts = SimulationSet(
+            settings=simulation_settings_1,
+            algorithm_set={
+                AlgorithmType.WEIGHT: [
+                    WeightAlgorithmConfig(),
+                ],
+                AlgorithmType.GROUP_MATCHER: [
+                    GroupMatcherAlgorithmConfig(
+                        csv_output_path=os.path.abspath(
+                            os.path.join(
+                                os.path.dirname(__file__),
+                                "../../../..",
+                                f"api/ai/external_algorithms/group_matcher_algorithm/group-matcher/inpData/{175}-generated.csv",
+                            )
+                        ),
+                        group_matcher_run_path=os.path.abspath(
+                            os.path.join(
+                                os.path.dirname(__file__),
+                                "../../../..",
+                                "api/ai/external_algorithms/group_matcher_algorithm/group-matcher/run.py",
+                            )
+                        ),
+                    ),
+                ],
+            },
+        ).run(num_runs=1)
```

Comment:
why is rhis one num_runs=1 and not =num_trials?

Reply:
This is because it was intended to only be run once on the real data to reproduce what an actual user would do

Reply:
But I can still change it and just have it default to one instead

Reply:
Oh wait this was 1 run because group matcher is deterministic, so the output never changes

Reply:
Ah okay